packages/victory-line/src/helper-methods.js

```diff
@@ -35,6 +35,7 @@ const getBaseProps = (props, fallbackProps) => {
   const modifiedProps = Helpers.modifyProps(props, fallbackProps, "line");
   props = assign({}, modifiedProps, getCalculatedValues(modifiedProps));
   const {
+    ariaLabel,
```

Comment:
ditto, no changes needed in this file.

Reply:
yep yep, just haven't had a chance to update this PR yet...will be up shortly!